pkg/networkmanager/networking.css

```diff
@@ -39,14 +39,14 @@
         padding: 0;
     }
 
-    .pf-c-card {
-        overflow-x: auto;
-    }
-
     #network-interface-hw, #network-interface-mac {
       font-size: var(--pf-global--FontSize--md);
       font-weight: var(--pf-global--FontWeight--normal);
     }
+
+    .network-interface-status > span {
```

Comment:
Why is there a `<span>`? I understand it's needed in the CSS because it's in the HTML... but why is it in the HTML?

Reply:
I believe that this `td` contains sometimes more thn this status string, and the span is used for separation of the elements.
In any case, I did not add it now of course.